conferences/forms.py

```diff
@@ -5,7 +5,19 @@
 from users.models import Organization
 
 
-class UserPreferencesForm(forms.ModelForm):
+class UserPreferencesBaseForm(forms.ModelForm):
+    def bus_queryset(self, instance=None):
```

Comment:
https://github.com/django/django/blob/master/django/forms/models.py#L282
Who's the master now? :P